go/vt/mysqlctl/tmutils/permissions.go

```diff
@@ -49,13 +50,16 @@ func NewUserPermission(fields []*querypb.Field, values []sqltypes.Value) *tablet
 		Privileges: make(map[string]string),
 	}
 	for i, field := range fields {
-		switch field.Name {
-		case "Host":
+		switch strings.ToLower(field.Name) {
```

Comment:
I added that because password_last_changed is not Password_Last_Changed, but user is User. Since MySQL is inconsistent, I figured it's simpler to be consistent on our side.